### PR TITLE
Add template package

### LINKED ISSUE #30: Template package
Tired of copy pasting other packages and removing bits of code. But.. don't want to create a cli to generate code just yet. Middle ground - commit a `template` package that we can use to copy paste that has just the required boilerplate

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #30

### CODE REVIEW

**Comment by joshxg** [comment]
Search for `${UPDATE}` in the template...?

Code gen for this through a cli in the future should be pretty easy

File: packages/template/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "serve": "11.3.2",
+    "typescript": "^4.3.5",
+    "webpack": "5.40.0",
+    "webpack-cli": "4.7.2",
+    "webpack-dev-server": "4.0.0-beta.3"
+  },
+  "scripts": {
+    "start": "webpack-cli serve",
+    "build": "webpack --mode production",
+    "serve": "serve dist -p 3006",
```
Comment (mark-prins) [inline packages/template/package.json:28]: `${UPDATE}` in here?

**Review by mark-prins — APPROVED** [review]
👍  cool - great idea. will think about the cli version too now 😉

**Comment by joshxg** [comment]
Will update in new PR 😁

**Pull request merged**